BrowserKit/Sources/Redux/State.swift

```diff
@@ -7,4 +7,10 @@ import Foundation
 /// Defines the entire application state including the UI state and any model state that you used in the app.
 /// This state is stored inside of the `Store`, then you have views and other subscribers that will get notified
 /// every single time that state updates into the entire app.
-public protocol StateType {}
+public protocol StateType {
+    /// Returns the state for a default action by clearing any transient state from previous one.
+    ///
+    /// All the state properties that have a default value into the initializer should be restore to default.
+    /// A default action could be the default case for a reducer switch case.
+    static func defaultActionState(from state: Self) -> Self
```

Comment:
I don't think the word action should be used here. The state and the action are two separate things, the action triggers changes in the state but the state does not belong to the action like this method implies. I think just defaultState is descriptive enough.

Reply:
Ok perfect thanks @OrlaM i agree i'll remove it